unit_tests/testLinearSolver_Multigrid/testLinearSolver_Multigrid.f90

```diff
@@ -74,13 +81,15 @@ PROGRAM testLinearSolver_Multigrid
 REGISTER_SUBTEST('testInit',testInit)
 #ifdef FUTILITY_HAVE_PETSC
 REGISTER_SUBTEST('testPreAllocPETScInterpMat',testPreAllocPETScInterpMat)
+#if ((PETSC_VERSION_MAJOR<=3) && (PETSC_VERSION_MINOR<8))
```

Comment:
Are these the write combination?  This makes 3.7 a weird one that will go into all `#if` sections

Reply:
understandable...basically the interface changed in 3.8 (and the routines this affects has an if >=8 check)..so it's "right" but obviously untested with 3.7 directly.

Reply:
yeah... just stuck out to me because a lot of your checks were > 3.6 and this is < 3.8.  So it was odd.  But I'm fine with it if that's what it needs to be.